Specs/Scene/ModelSpec.js

```diff
@@ -1468,6 +1410,46 @@ defineSuite([
         });
     });
 
+    it('Loads with incrementallyLoadTextures set to true', function() {
+        return loadModelJson(texturedBoxModel.gltf, {
+            incrementallyLoadTextures : true
+        }).then(function(m) {
+            // Get the rendered color of the model before textures are loaded
+            var loadedColor = verifyRender(m);
+
+            pollToPromise(function() {
+                // Render scene to progressively load textures
+                scene.renderForSpecs();
+                // Textures have finished loading
+                return !defined(m._loadResources);
```

Comment:
It would be OK to have a readonly property that is `pendingTextureRequests`, which would be zero once all textures are loaded.